src/decisionengine/framework/config/ChannelConfigHandler.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, global_config, channel_config_dir):
         self.channel_config_dir = channel_config_dir
         self.channels = {}
         self.logger = _make_de_logger(global_config)
-        self.logger = self.logger.bind(module=__name__.split(".")[-1])
+        self.logger = self.logger.bind(module=__name__.split(".")[-1], channel="dechannel")
```

Comment:
Probably `"dechannel"` should go in a constant. And maybe "engine" instead of "dechannel"?

Reply:
Got it. Made the changes.